kolibri/content/utils/annotation.py

```diff
@@ -36,7 +38,41 @@ def update_channel_metadata():
     for channel_id in channel_ids:
         if not ChannelMetadata.objects.filter(id=channel_id).exists():
             import_channel_from_local_db(channel_id)
-            set_availability(channel_id)
+    fix_multiple_trees_with_id_one()
```

Comment:
Minor Q: the docstring says

> If we are potentially moving from a version of Kolibri that did not import its content data

However, now this function seems a bit broader in scope, so might want to tweak the comment a bit?

Reply:
True, true. Can update shortly.